src/convertToObject.js

```diff
@@ -1,18 +1,24 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';\n')
+    .reduce((acc, cssStyle) => {
+      const [key, value] = cssStyle.split(':').map(part => part.trim());
```

Comment:
<img width="1049" alt="image" src="https://github.com/mate-academy/js_style-to-object/assets/86789161/d2311e87-5b46-4087-a0fc-c63e1909b18d">